xload/type/endpoint.go

```diff
@@ -0,0 +1,32 @@
+package xloadtype
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+)
+
+type Endpoint struct {
```

Comment:
Can we extend the URL from the standard lib instead of creating a new type

Reply:
Let me try that, however, do you see things like path, query params being used?
I can imagine path being useful.

Reply:
Decided to keep `Endpoint` as well, but also added an aliased type for `url.URL` type.